src/openforms/js/components/admin/forms/Select.js

```diff
@@ -25,7 +31,7 @@ const Select = ({
 
   const prefix = useContext(PrefixContext);
   name = prefix ? `${prefix}-${name}` : name;
-  const options = allowBlank ? [BLANK_OPTION].concat(choices) : choices;
+  const options = allowBlank ? BLANK_OPTION.concat(choices) : choices;
```

Comment:
Does this not cause problems with the 'original' `BLANK_OPTION` being edited?

Reply:
It is only used in this file and not imported anywhere :+1: